README.arm.md

```diff
@@ -129,3 +129,43 @@ finding appropriate `crouton` help.
 
 On the current [Scaleway](http://scaleway.com) ARM servers, the Julia
 build works out of the box.
+
+## nVidia Jetson TX2
+
+Julia builds and runs on the [nVidia Jetson TX2](http://www.nvidia.com/object/embedded-systems-dev-kits-modules.html) platform with minimal configuration changes. A full multi-threaded build, including LLVM, will complete in around two hours. All tests pass and CUDA functionality is available through, e.g., [CUDAdrv](https://github.com/JuliaGPU/CUDAdrv.jl). 
```

Comment:
I took the liberty of pushing a commit directly to master to fix this, as it's terribly annoying for anybody having automated checks enabled (can't even commit something locally...).